backend/cfg/cfg_dataflow.mli

```diff
@@ -57,14 +57,20 @@ module Forward
 module type Backward_domain = sig
   type t
 
+  (** Value to use if a block didn't have any other assigned (similar to the
+      [init] argument in [run]) *)
   val bot : t
 
   val compare : t -> t -> int
   (* note: `compare` is an order that can be passed to e.g. `Map.Make` or
      `Set.Make`; it does not need to be compatible with `less_than`. *)
 
+  (** [join old_value successor_value] has to join last know value for the end
+      of the block with an updated value from a successor. *)
   val join : t -> t -> t
 
+  (** [less_equal new_value old_value] should return [true] if there is no need
```

Comment:
I believe this comment should be verbose because otherwise someone can pass e.g. the default comparison for sets which doesn't work here. Also it's confusing because we always have [new_value >= old_value] because [new_value = join incoming old_value].

Reply:
Changed the comments.